auto_py_to_exe/web/js/i18n.js

```diff
@@ -18,6 +19,24 @@ const translationMap = {
                 gr: 'Διαδρομή Αρχείου',
                 ar: 'موقع البرنامج النصي',
             },
+            language: {
+                en: 'Language ',
+                zh: '成语 ',
+                zh_tw: '成語 ',
+                ru: 'идиома ',
+                de: 'Idiom ',
+                pt_br: 'Idioma ',
+                sp: 'Idioma ',
+                sp_la: 'Idioma ',
+                tr: 'Deyim ',
+                th: 'สำนวน ',
+                fr: 'Langage ',
+                it: 'Idioma ',
+                id: 'Idiom ',
+                cz: 'Idiom ',
+                gr: 'ιδίωμα ',
+                ar: 'لغة. مثل',
+            },

```

Comment:
Are these related to a change in index.html?

Reply:
Yes, because when you change the language the text "Language"  in the UI doesnt change to the one related

Reply:
I see - any need for the spaces at the end of each of these strings?

Reply:
For a little of space between the language drop down list, nothing important

Reply:
Before we had "Language:" but now the colon (`:`) isn't showing - could we get that back? Preferably keeping the colon out of the translation file and in the index.html (after the `small` tag with the translation on it.

Also removing the spaces from here would be fine - if spacing is needed in the UI (there should be natural spacing due to the spacing between tags), you can add a 4px left margin to the dropdown in CSS.

If you're not sure about any of this or want me to help - let me know 👌

Reply:
Ok, Everything done!